CHANGELOG.rst

```diff
@@ -2,6 +2,79 @@
 SDK Changelog
 =============
 
+2.5.0-alpha.1 (2024-11-26)
+==========================
+
+Second preview release of the upcoming 2.5.0 version.
+
+New features
+------------
+
+* [#4320] Made it possible to let the backend drive the content of the confirmation page more.
+* [#4320] Moved cosign stories to shared location.
+* [#4320] Updated introduction text for cosigning.
+* [#4320] Support optional code for cosign login URL.
+* [#4320] Only display cosign login options if links in emails are not allowed.
+* [#4320] Set up dedicated route for cosign start.
+* [#4546] Added alert styling to the soft required errors component.
+* [#4546] Ensure that soft required validation errors and input are linked in an accessible way.
+* [#4546] Implemented conditional error display.
+* [#4546] Render soft required errors - static variant.
+* [#4718] Re-calculate the "verticalSpaceUsed" on resize.
+* [#4718] Bumped design tokens to 0.55.0.
+* Ran browserslist upgrade.
+* [#4718] Improved accessibility for increased zoom level.
+* [#4720] Moved accessible link text to aria-label.
+* [#4717] Use different ErrorMessage roles for info and ok modifiers.
+* [#4720] Added form name to the title of the form step pages. 
+* [#4720] Self explaining links.
+* [#4720] Made deprecated-select and progress indicator elements focusable.
+* [#4717] Added role alert to ErrorMessage.
+* [#4717] Added text alternative to loader.
+* [#4717] Use button element for the Modal close button.
+* [#4716] Added jest test for accessible form field validation behaviour.
+* [#4716] Added textual alternative to multiple value row delete button.
+* [#4716] Improved File upload template.
+* [#4716] Replaced DOM query with ref.
+* [#4716] Made default ValidationErrors id undefined.
+* [#4716] Reverted file table back to list.
+* [#4716] Limited accessibility changes for most components to only non-multiple instances.
+* [#4716] Fixed some storybook tests.
+* [#4716] Added more explicit propTypes and error message ids.
+* [#4716] Added autocomplete property to EmailField.
+* [#4716] Linked error message to file upload and turned upload list into a table.
+* [#4716] Added aria attributes to invalid form fields.
+* [#4716] Marked error messages with a descriptive id.
+* [#4716] Connected input to error message using aria-describedby.
+* [#4716] Show the current form step in progress indicator.
+
+Bugfixes
+--------
+
+* [#4827] Do not add initial_data_reference if empty.
+* [#4546] Properly pass initial_data_reference to the nextUrl query params on the login URL.
+
+Project maintenance
+-------------------
+
+* Ensure prettier checks jsx files.
+* [#4849] Ensure .jsx files are picked up for translation as well.
+* [#3283] Removed password Formio component.
+* Fixed button stories/documentation page.
+* [#4320] Updated stories and UX of completion view.
+* Excluded file renames from git blame.
```

Comment:
```suggestion
```
kind of implied by the JSX renames, it just has to be a separate commit otherwise `git-blame-ignore-revs` doesn't work properly.